src/coprocessor/tracker.rs

```diff
@@ -221,11 +221,8 @@ impl Tracker {
                 .inc_by(self.total_perf_stats.block_read_byte as i64);
         });
 
-        tls_collect_cf_stats(
```

Comment:
It seems that function "tls_collect_cf_stats" is only called here. Maybe we could remove it?

Reply:
Do you mean inline the content manually?

Reply:
Sorry,  I got it wrong. The code of this function has been deleted.